conf/types.json

```diff
@@ -0,0 +1,66 @@
+{
+  "carbonblack":{
+    "userName": ["username"],
+    "destinationDomain": ["domain"],
+    "processPath": ["parent_path", "process_path", "path"],
+    "filePath": ["path"],
+    "transportProtocol": ["protocol"],
+    "processName": ["parent_name", "process_name"],
+    "fileName": ["observed_filename", "file_path"],
+    "command": ["cmdline"],
+    "fileHash": ["process_md5", "parent_md5", "expect_followon_w_md5", "md5"],
+    "deviceAddress": ["interface_ip", "comms_ip"],
+    "sourceAddress": ["ipv4", "local_ip"],
+    "destinationAddress": ["remote_ip"],
+    "sourcePort": ["port", "local_port"],
+    "destinationPort": ["remote_port"]
+  },
+  "cloudwatch":{
+    "userName": ["userName", "owner", "invokedBy"],
+    "sourceAccount": ["account"],
+    "destinationAccount": ["recipientAccountId"],
+    "transportProtocol": ["protocol"],
+    "eventType": ["eventType"],
+    "eventName": ["eventName"],
+    "region": ["region"],
+    "userAgent": ["userAgent"],
+    "sourceAddress": ["source", "sourceIPAddress"],
+    "destinationAddress": ["destination"],
+    "sourcePort": ["srcport"],
+    "destinationPort": ["destport"]
+  },
+  "cloudtrail": {
+    "sourceAccount": ["account", "accountId"],
+    "destinationAccount": ["recipientAccountId"],
+    "eventType": ["eventType"],
+    "eventName": ["eventName"],
+    "region": ["region", "awsRegion"],
+    "userAgent": ["userAgent"],
+    "sourceAddress": ["sourceIPAddress"]
+  },
+  "ghe": {
+    "userName": ["current_user"],
+    "destinationAddress": ["remote_address"],
+    "sourcePort": ["port"]
+  },
+  "osquery": {
```

Comment:
add `"fileHash": ["md5", "sha1", "sha256"]` (see https://osquery.io/docs/tables/#hash)

Reply:
add `sourceUserId`, have the array contain `uid` (see https://osquery.io/docs/tables/#users)

Reply:
add `receiptTime`, have the array contain `unixTime` (all osquery logs have this field, it denotes when the info was collected)

Reply:
add `fileSize`, have it contain `size`. ex: https://osquery.io/docs/tables/#file_events